Problem building on Ubuntu 15.10

When I run make I get the following message:

```
$ make

  CC      build-x86_64/src/db.o
src/db.c:214:32: warning: ‘struct timespec’ declared inside parameter list
 static void gettimespec(struct timespec *ts, uint32_t offset)
                                ^
src/db.c: In function ‘gettimespec’:
src/db.c:220:4: error: dereferencing pointer to incomplete type ‘struct timespec’
  ts->tv_sec  = tv.tv_sec + offset;
    ^
src/db.c: In function ‘database_thread’:
src/db.c:227:18: error: storage size of ‘ts’ isn’t known
  struct timespec ts;
                  ^
src/db.c:227:18: warning: unused variable ‘ts’ [-Wunused-variable]
Makefile:108: recipe for target 'build-x86_64/src/db.o' failed
make: *** [build-x86_64/src/db.o] Error 1
```

Am I missing something?

--- a/Makefile
+++ b/Makefile
@@ -61,7 +61,7 @@ ifeq ($(LIBDIR),)
 LIBDIR  := $(PREFIX)/lib
 endif
 MOD_PATH:= $(LIBDIR)/$(PROJECT)/modules
-CFLAGS	+= -I$(LIBRE_INC) -Iinclude
+CFLAGS	+= -I$(LIBRE_INC) -Iinclude -std=gnu99
 BIN	:= $(PROJECT)$(BIN_SUFFIX)
 MOD_BINS:= $(patsubst %,%.so,$(MODULES))
 APP_MK	:= src/srcs.mk

